pyfolio/tests/test_perf_attrib.py

```diff
@@ -152,3 +152,78 @@ def test_perf_attrib_simple(self):
 
         pd.util.testing.assert_frame_equal(expected_exposures_portfolio,
                                            exposures_portfolio)
+
+    def test_perf_attrib_regression(self):
+
+        positions = pd.read_csv('pyfolio/tests/test_data/positions.csv',
+                                index_col=0, parse_dates=True)
+
+        positions.columns = [int(col) if col != 'cash' else col
+                             for col in positions.columns]
```

Comment:
Why is this needed?

Reply:
the columns of `positions` get read in from this csv as all strings (besides the cash column), but in other dataframes, the columns get read in as ints, so multiplying the dfs together fails. Just standardizing them